backend/development.ini.sample

```diff
@@ -376,6 +376,12 @@ api.key = %(basic_setup.api_key)s
 ## TLS usage to communicate with your LDAP server
 ; ldap_tls = False
 
+## Read only fields per auth_type, parameter name is user.profile.read_only_fields.{auth_type}
+# value is a comma separated list of read-only fields.
```

Comment:
"fields" are database table's columns?
if yes, maybe it requires in line in the doc, if it's consistent with other parameter's doc

Reply:
yes fields are database table's columns. But it is more the form fields you see on the UI when editing your (or another's user) profile. I don't really know how to name them in this case...

Reply:
"fields" is fine. What I'm pointing at is how an admin will be able to know which values fields can take

Reply:
Ok I see, I will make it more clear then

Reply:
resolved in last commit